consolidate_metrics_.py

```diff
@@ -0,0 +1,48 @@
+
+import os
+import argparse
+import pandas as pd
+
+def consolidate_metrics(input_folder, output_csv_path):
+    # Initialize an empty dictionary to hold DataFrames for each AI model
+    model_data_dict = {}
+    csv_files = [f for f in os.listdir(input_folder) if f.endswith('.csv')]
+    
+    # Loop through each CSV file and read it into a DataFrame
+    for i, csv_file in enumerate(csv_files):
+        model_name = f"Model_{i+1}"
```

Comment:
Maybe we should try to keep the original model names?

Reply:
Yes, @pdc878 I thought those model names were just for our project. We should not assume that everybody will name their models that way. Can you make that change and update the pull request?